yoti/YotiHelper.php

```diff
@@ -91,54 +93,49 @@ public function link($currentUser = NULL) {
           self::SDK_IDENTIFIER
       );
       $activityDetails = $yotiClient->getActivityDetails($token);
+      $profile = $activityDetails->getProfile();
     }
     catch (Exception $e) {
       self::setFlash('Yoti could not successfully connect to your account.', 'error');
 
       return FALSE;
     }
 
-    // If unsuccessful then bail.
-    if ($yotiClient->getOutcome() !== YotiClient::OUTCOME_SUCCESS) {
-      self::setFlash('Yoti could not successfully connect to your account.', 'error');
-
-      return FALSE;
-    }
-
-    if (!$this->passedAgeVerification($activityDetails)) {
+    if (!$this->passedAgeVerification($profile)) {
+      self::setFlash("Could not log you in as you haven't passed the age verification", 'error');
       return FALSE;
     }
 
     // Check if Yoti user exists.
-    $drupalYotiUid = $this->getDrupalUid($activityDetails->getUserId());
+    $drupalUid = $this->getDrupalUid($activityDetails->getRememberMeId());
```

Comment:
How come we changed this from `drupalYotiUid` to `drupalUid`?

Reply:
Yes this the Drupal User Id, so it's clearer this way. It's just the naming.